includes/admin_settings_tab.php

```diff
@@ -0,0 +1,333 @@
+							<?php } ?>
+						</div>
+					</form>
+				</div>
+							
+				<div class="settings-card">
+					<h3><strong><?php _e('Address Geocoding', '12-step-meeting-list') ?></strong></h3>
+					<p><?php _e('Code4Recovery is working on a new method for geocoding addresses. You can decide whether to use the old way, or whether to help us test the new way.', '12-step-meeting-list') ?></p>
+				<form method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+					<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+					<select name="tsml_geocoding_method" onchange="this.form.submit()">
+						<option value="legacy" <?php selected($tsml_geocoding_method, 'legacy') ?>>
+							<?php _e('Legacy Method', '12-step-meeting-list') ?>
+						</option>
+						<?php if (!empty($tsml_google_maps_key)) { ?>
+							<option value="google_key" <?php selected($tsml_geocoding_method, 'google_key') ?>>
+								<?php _e('Use my Google API Key', '12-step-meeting-list') ?>
+							</option>
+						<?php } ?>
+						<option value="api_gateway" <?php selected($tsml_geocoding_method, 'api_gateway') ?>>
+							<?php _e('BETA - API Gateway - BETA', '12-step-meeting-list') ?>
+						</option>
+					</select>
+				</form>
+				<?php if (!empty($tsml_google_maps_key)) { ?>
+					<p>If you select "Use my Google API Key", then you <strong>must</strong> go into the Google Console and enable the geocode API for your key</p>
+				<?php } ?>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div id="column3" class="col one-third">
+		<!-- Put About Us here -->
+		<div class="postbox" >
+			<div class="inside">
+				<h1><?php _e('About Us', '12-step-meeting-list') ?></h1>
+				<div class="settings-card">
+					<div class="c4r-logo" style="float:right; margin-top:-55px; margin-right: -10px;" >
+						<a href="https://code4recovery.org"><img src="/wp-content/plugins/12-step-meeting-list/assets/img/code4recovery.svg" alt="Code For Recovery" ></a>										
+					</div>
+					<div>
+						<p><?php _e('This <b>12 Step Meeting List</b> plugin (TSML) is one of the free services offered by the nonprofit organization <b>Code For Recovery</b> whose volunteer members build and maintain technology services for recovery fellowships such as AA and Al-Anon.', 
+						'12-step-meeting-list') ?></p>										
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="postbox">
+		<!-- Put Need Help here -->
+			<div class="inside">
+				<h1><?php _e('Need Help?', '12-step-meeting-list') ?></h1>
+				<div class="settings-card">
+									
+					<p><?php _e("To get information about this product or our organization, simply use one of the linked buttons below. Both the plugin Wiki page and/or the <b>Code For Recovery</b> website are great sources of information. 
+					You can also ask questions directly throght our GitHub discussion forum which is monitored daily by members of our maintenance team.", '12-step-meeting-list') ?></p>
```

Comment:
there's a typo on the word "through" but i recommend cutting this paragraph, i don't think it adds anything that the user wouldn't surmise from the links themselves.

Reply:
👍